crates/bevy_window/src/window.rs

```diff
@@ -329,6 +337,14 @@ impl Window {
         self.resolution.physical_height()
     }
 
+    /// The window's client size in physical pixels, with x being width and y being height.
```

Comment:
```suggestion
    /// The window's client size in physical pixels.
```

Just a nit and others might disagree, but I can't think of a case in the engine where a `Vec2`'s `x` field refers to height or something. It seems self-documenting to me so I think it would be okay to leave this part out.